contrib/local.mk.example

```diff
@@ -28,8 +24,24 @@
 #
 # CMAKE_BUILD_TYPE := Debug
 
-# By default, all libraries are statically linked to nvim. Uncomment these
-# entries to use dynamic linking instead.
+# By default, nvim uses bundled versions of its required third-party
+# dependencies.
+# Uncomment these entries to instead use system-wide installations of
+# them.
+#
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_BUSTED=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_DEPS=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_JEMALLOC=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LIBTERMKEY=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LIBUV=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LIBVTERM=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LUAJIT=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LUAROCKS=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_MSGPACK=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_UNIBILIUM=OFF
+
+# By default, all libraries are statically linked to nvim.
+# Uncomment these entries to instead use dynamic linking.
```

Comment:
> But I don't think there's a use case for that (?)

I don't think so either.  Some may have RPATH issues and expect to be installed into a known location (and you probably don't want them to live in the .deps area in that case).  So if you really want dynamic linking, then you really want to get the dependencies in place and not use the bundled versions, IMO.